client/src/components/Admin/Country/index.vue

```diff
@@ -21,6 +22,10 @@
                     <td>{{ countryDetails.name }}</td>
                     <td>{{ countryDetails.description }}</td>
                     <td>{{ countryDetails.countryCode }}</td>
+                    <td> <b-icon :icon="handleVisibilty(countryDetails.isBlocked)" aria-hidden="true" font-scale="2"
+                            @click="toggleHide(countryDetails.isBlocked, countryDetails.id)" style="cursor:pointer"
+                            variant="success">
```

Comment:
`variant="success"` its fine but would be nice if you make it dynamic
like `secondary` on `countryDetails.isBlocked = false` and
 `success` on `countryDetails.isBlocked = true`